haas/cli.py

```diff
@@ -27,6 +27,21 @@
 
 from functools import wraps
 
+## Hook to the client library
+from haas.client.auth import *
+from haas.client.client import Client
+
+
+ep = "http://127.0.0.1:5000" or os.environ.get('HAAS_ENDPOINT')
```

Comment:
a) This should be in the client library, and
b) we shouldn't have defaults for these. That could cause unexpected surprises!